frontend/invest/src/styles/tokens.css

```
@@ -0,0 +1,222 @@
+  /* Subtle — these are secondary identifiers, not theme colors.
+     Backgrounds are very low-saturation tints; foregrounds sit
+     just dark enough to read at 11–12px. */
+  --pill-kis-bg:    #eaf2ff; --pill-kis-fg:    #2a6df4;  /* KIS — small blue */
+  --pill-upbit-bg:  #e8f3ec; --pill-upbit-fg:  #0e8a5f;  /* Upbit — small green */
+  --pill-toss-bg:   #efebf8; --pill-toss-fg:   #5b3aa8;  /* Toss / manual — soft purple */
+  --pill-isa-bg:    #f1ede1; --pill-isa-fg:    #7a6a3a;  /* ISA — muted olive */
+  --pill-pension-bg:#ecedef; --pill-pension-fg:#5f6470;  /* Pension — stone */
+  --pill-paper-bg:  #eef0f3; --pill-paper-fg:  #4e5968;  /* Paper / mock */
+
+  /* ---------------- Type: Family ------------------------ */
+  --font-sans: "Pretendard Variable", Pretendard,
```

Stylelint (17.10.0)

[error] Expected "Pretendard" to be "pretendard" (value-keyword-case)

(value-keyword-case)